File: app/src/main/java/com/jml/rappichallenge/models/tools/LanguageHelper.java
package com.jml.rappichallenge.models.tools;

import com.jml.rappichallenge.R;

import androidx.annotation.StringRes;

public class LanguageHelper
{
	public static @StringRes int GetLanguageResNameByCode(String code)
	{
		// TODO usar locale para esto?
		switch (code)
		{
			case "en":
				return R.string.lan_english;
			default:
				return 0;
		}
	}
}

File: app/src/main/java/com/jml/rappichallenge/models/tools/LanguageHelper.kt
package com.jml.rappichallenge.models.tools

import com.jml.rappichallenge.models.enums.Language
import java.util.*

object LanguageHelper {

    fun getLanguagByCode(code: String): Language {

        return when (code) {
            "es" -> Language.Spanish
            "en" -> Language.English
            else -> Language.English
        }
    }

    val userLanaguage : Language
        get() = getLanguagByCode(Locale.getDefault().language)
}
